src/js/models/Pregunta.js

```diff
@@ -0,0 +1,13 @@
+import {createQuestion} from "../view/createQuestion"
+export class Pregunta {
+    constructor(respuesta,questions){
+        let result = document.querySelector('.respuesta')
+        result.textContent= respuesta
+        result.addEventListener('click',()=>{
+            let question = document.querySelector('.pregunta1')
```

Comment:
when a variable isn't reassigned you must use const, some things like linters make you aware of that and give you suggestions you could try to integrate eslint in your project we will configure it this weekend.

Reply:
done